x/incentives/keeper/distribute_test.go

```diff
@@ -166,7 +165,7 @@ func (s *KeeperTestSuite) TestDistribute() {
 	}
 }
 
-func (s *KeeperTestSuite) TestDistributeToConcentratedLiquidityPools() {
+func (s *KeeperTestSuite) TestDistribute_InternalIncentives_ConcentratedPool() {
```

Comment:
Note: many of the tests here are obsolete. Attempted to simplify and make sense out of it by removing the redundant or obsolete test cases